mne/source_estimate.py

```diff
@@ -335,18 +338,43 @@ def read_source_estimate(fname, subject=None):
 
     if ftype == 'volume':
         stc = VolSourceEstimate(**kwargs)
+    elif ftype == 'h5' and kwargs['data'].ndim == 3:
+        stc = VectorSourceEstimate(**kwargs)
     else:
         stc = SourceEstimate(**kwargs)
 
     return stc
 
 
-def _make_stc(data, vertices, tmin=None, tstep=None, subject=None):
-    """Generate a surface, volume or mixed source estimate."""
+def _make_stc(data, vertices, tmin=None, tstep=None, subject=None,
+              vector=False, source_nn=None):
+    """Helper function to generate a surface, vector-surface, volume or mixed
```

Comment:
"""Generate a surface, ..."""

imperative voice, helper-ness is implied by the leading `_`